grader/assignments/fork-wait/invalid-address.c

```diff
@@ -0,0 +1,44 @@
+uint64_t* VIRTUALMEMORYSIZE = (uint64_t*) 4294967296; // 4GB of virtual memory
+uint64_t* ELF_ENTRY_POINT = (uint64_t*) 65536; // = 0x10000 (address of beginning of code)
+
+
+// computes the exit code from the status value
+// exit code is in the least significant bits from 8 to 16
+uint64_t wexitstatus(uint64_t status) {
+  return status * 281474976710656 / 72057594037927936;
+}
+
+int main() {
+  uint64_t sum;
+  uint64_t pid;
+  uint64_t* code;
+
+  code = malloc(8);
```

Comment:
The variable name is misleading. Something like `heap_top` would probably fit better.